src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -159,53 +159,55 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
     public PageReference enableDataImportFieldMapping() {
         STG_SettingsManager_CTRL.idPanelCurrent = idPanel();
 
+        // Check the deployment status and set toggleDisabled value
         deployment = checkDeploymentStatus();
 
         if (toggleDisabled == false || toggleDisabled == null) {
```

Comment:
I think toggleDisabled should be changed to be a positive, and more descriptive to what it actually represents.  The actual toggle on the page UI can respond to the attribute value, which should be called something like `hasDeploymentInProgress`